tools/python/util/check_onnx_model_mobile_usability.py

```diff
@@ -43,24 +53,38 @@ def check_usability():
 
     try_eps = usability_checker.analyze_model(args.model_path, skip_optimize=False, logger=logger)
-    check_model_can_use_ort_mobile_pkg.run_check(args.model_path, args.config_path, logger)
 
-    logger.info(
-        "Run `python -m onnxruntime.tools.convert_onnx_models_to_ort ...` to convert the ONNX model to ORT "
-        "format. "
-        "By default, the conversion tool will create an ORT format model with saved optimizations which can "
-        "potentially be applied at runtime (with a .with_runtime_opt.ort file extension) for use with NNAPI "
-        "or CoreML, and a fully optimized ORT format model (with a .ort file extension) for use with the CPU "
-        "EP."
-    )
-    if try_eps:
-        logger.info(
-            "As NNAPI or CoreML may provide benefits with this model it is recommended to compare the "
-            "performance of the <model>.with_runtime_opt.ort model using the NNAPI EP on Android, and the "
-            "CoreML EP on iOS, against the performance of the <model>.ort model using the CPU EP."
-        )
+    if args.check_mobile_package:
+        can_use_mobile_package = check_model_can_use_ort_mobile_pkg.run_check(args.model_path, args.config_path, logger)
+
+        if can_use_mobile_package:
+            logger.info(
+                "The mobile package requires an ORT format model.\n"
+                "Run `python -m onnxruntime.tools.convert_onnx_models_to_ort ...` to convert the ONNX model to ORT "
+                "format.\n"
+                "By default, the conversion tool will create an ORT format model with saved optimizations which can "
+                "potentially be applied at runtime (with a .with_runtime_opt.ort file extension) for use with NNAPI "
+                "or CoreML, and a fully optimized ORT format model (with a .ort file extension) for use with the CPU "
+                "EP."
+            )
+
+            if try_eps:
+                logger.info(
+                    "As NNAPI or CoreML may provide benefits with this model it is recommended to compare the "
+                    "performance of the <model>.with_runtime_opt.ort model using the NNAPI EP on Android, and the "
+                    "CoreML EP on iOS, against the performance of the <model>.ort model using the CPU EP."
+                )
+            else:
+                logger.info("For optimal performance the model should be used with the CPU EP. ")
     else:
-        logger.info("For optimal performance the <model>.ort model should be used with the CPU EP. ")
+        if try_eps:
+            logger.info(
+                "As NNAPI or CoreML may provide benefits with this model it is recommended to compare the "
+                "performance of the model using the NNAPI EP on Android, and the CoreML EP on iOS, "
+                "against the performance using the CPU EP."
+            )
+        else:
+            logger.info("For optimal performance the model should be used with the CPU EP. ")
```

Comment:
I wonder if it makes more sense to limit the `--log_level` option to info or verbose. is there a reason to run these scripts with a higher log level if the useful output is at info?